src/main/scala/rocket/SimpleHellaCacheIF.scala

```diff
@@ -131,6 +132,7 @@ class SimpleHellaCacheIF(implicit p: Parameters) extends Module
   replayq.io.nack.bits := s2_req_tag
   replayq.io.resp := io.cache.resp
   io.requestor.resp := io.cache.resp
+  io <> DontCare
```

Comment:
I still have no idea why porting to chisel3 has different connection semantic.

Reply:
There are some IOs that are not connected which is found in CI test 7 (log is shown below). In Chisel 2, the compile options is `chisel3.core.ExplicitCompileOptions.NotStrict` and whether it means the IO initialization is not checked before?

```text
Exception in thread "main" firrtl.passes.PassExceptions: 
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.perf.canAcceptStoreThenRMW <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.perf.tlbMiss <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.s2_xcpt.ae.ld <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.perf.blocked <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.s2_nack <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.s2_gpa <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.cache.keep_clock_enabled <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.clock_enabled <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.perf.release <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.perf.storeBufferEmptyAfterStore <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.s2_xcpt.gf.ld <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.s2_xcpt.ma.ld <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.s2_xcpt.ae.st <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.s2_xcpt.gf.st <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.s2_nack_cause_raw <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.s2_xcpt.ma.st <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.cache.s1_data.mask <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.s2_xcpt.pf.ld <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.perf.grant <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.perf.canAcceptStoreThenLoad <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.s2_uncached <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.s2_xcpt.pf.st <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.perf.storeBufferEmptyAfterLoad <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.ordered <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.perf.acquire <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.s2_gpa_is_pte <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.perf.canAcceptLoadThenLoad <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.replay_next <= VOID
firrtl.passes.CheckInitialization$RefNotInitializedException:  : [module SimpleHellaCacheIF]  Reference io is not fully initialized.
   : io.requestor.s2_paddr <= VOID
firrtl.passes.PassException: 29 errors detected!
```